packages/geo/src/lib/query/shared/query.directive.ts

```diff
@@ -155,6 +158,10 @@ export class QueryDirective implements AfterViewInit, OnDestroy {
     }
 
     const resolution = this.map.ol.getView().getResolution();
+
+    this.setQueryableForCombineWMS_WFS(this.map.layers);
```

Comment:
Maybe it would be cleaner and more simple to just adapt the layerIsQueryable condition ? Instead of rewrite each time the layer queryable attribute ?

Reply:
I don't see that property in vector layer... @PhilippeLafreniere18  where is it?
![image](https://user-images.githubusercontent.com/32003632/144880168-eca6c346-0793-4abe-b860-c7347bf5aa15.png)

Reply:
In geo/.../query,utils (https://github.com/infra-geo-ouverte/igo2-lib/blob/master/packages/geo/src/lib/query/shared/query.utils.ts#L12)